### ISSUE TITLE
BUG:  Missing whentime and clear the form each time I try to post

### ISSUE THREAD

**Issue by shellaaaron** [krisives/cronnit.us#25]
I just hope this in active development I am testing the site and got the error "  Missing whentime  " but actually time is entered.

and it clear the form with all data, it shouldn't clear the form if there is error as well.

**Comment by krisives** [comment]
Yes, Cronnit is actively maintained.

What browser are you using?

Have you tried testing with all browser plugins/extensions disabled?

This specific error is given when the `whentime` variable is not provided when submitting the form. The form itself is very simple:

https://github.com/krisives/cronnit.us/blob/ec5be288722744582c7fb5a68122701f6e79bf61/templates/pages/submit.html#L26-L29

It's also possible your browser is doing something strange when it sees a `type="time"` style of input element. Does your browser render the input element as a time selection with a colon between the hour/minute?

**Comment by shellaaaron** [comment]
No, actually I am using the site to test it https://cronnit.us/, I am using firefox.

It's clear the form each time and it show this error.

" Missing whentime "

**Comment by krisives** [comment]
Can you verify it happens using Firefox when disabling all of your browser add-ons?

**Comment by krisives** [comment]
You can quickly test by going to Help menu in Firefox and selecting Restart with Addons Disabled.

**Comment by shellaaaron** [comment]
I was able to use CSV to submit a few posts to reddit by using the site https://cronnit.us, and then got this error
Reddit had a problem with this post: HTTP error: Client error: `POST https://oauth.reddit.com/api/submit` resulted in a `401 Unauthorized` response: {"message": "Unauthorized", "error": 401}

**Comment by krisives** [comment]
This means that the token Reddit provided to us was not valid. From what I can tell your browser or a proxy in between is significantly violating the HTTP specification. There isn't much Cronnit can do to fix this.

**Comment by ShawkMusic** [comment]
I was having this problem because I thought the time format was hh:mm ss so I was trying to type numbers once I reached the 3rd number (that i thought was seconds) when in reality the format is hh:mm a.m./p.m.

Try typing the hours, then minutes, then type an "a" for AM or "p" for PM

Someone should probably make the time format clearer. As a Firefox user, it didn't give me an interface for selecting the time, it only let me type it out. It was only when I tried it on Chrome that I was able to use the time setting interface and realize the format was hh:mm am/pm instead of hh:mm ss
